thrifty/cost_variance.sp

```diff
@@ -15,7 +21,8 @@ benchmark "cost_variance" {
   description   = "."
   documentation = file("./thrifty/docs/cost_variance.md")
   children = [
-    control.full_month_cost_changes
+    control.full_month_cost_changes,
+    control.full_month_forecast_cost_changes
```

Comment:
```suggestion
    control.cost_explorer_full_month_cost_changes,
    control.cost_explorer_full_month_forecast_cost_changes
```

I think this is the service we use in the queries. Similar to other query renames, we should include the service name in the control names